### ISSUE TITLE
Payload data is nested by extra layer

### ISSUE THREAD

**Issue by atomdmac** [mautic/mautic#5119]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | X
| Feature request? | 
| Enhancement? | 

## Description:
Somewhere between version `2.8.2` and  `2.9.1` the `mautic.lead_post_save_update` test webhook payload seems to have changed to introduce an additional layer of `mautic.lead_post_save_update`

-I'm not currently sure if this applies to test payloads _and_ normal "production" payloads or not.  I've only tested it with the `Send Test Payload` button at this point.-

This appears to be an issue with the test payloads (sent from `Settings > Webhooks > [Webhook] > Edit > Send Test Payload`).  Normal payloads (so far) seem to unaffected.

### As of 2.8.2
```
{ 'mautic.lead_post_save_update':
   { lead:
      { ...lead data here },
  timestamp: '2017-10-09T11:07:44-04:00' }
```
### As of 2.9.1
```
{ 'mautic.lead_post_save_update':
   { 'mautic.lead_post_save_update':
      [ { lead:
           { ...lead data here} ] },
  timestamp: '2017-10-09T15:11:28+00:00' }
```

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 2.9.1
| PHP version | 7.0.18-0ubuntu0.16.04.1

### Steps to reproduce:
1. Create a new 'Contact Updated Event' webhook
2. Send a test payload
3. Notice that there is an extra layer of `mautic.lead_post_save_update`
 
### Log errors: 

None that I was able to find in the `system settings` > `logs` area.

**Comment by atomdmac** [comment]
Update: This still exists in `2.10.1`.

**Comment by escopecz** [comment]
Cannot replicate. Can you please confirm it's still an issue?

**Comment by Clement134** [comment]
This issue still exists in `2.15.0-dev`.

Event name is [automatically added](https://github.com/mautic/mautic/blob/staging/app/bundles/WebhookBundle/Controller/AjaxController.php#L133) on test payload, but it was also present in the fake payload for some lead events ([lead_post_save_new](https://github.com/mautic/mautic/blob/staging/app/bundles/LeadBundle/Assets/WebhookPayload/lead_post_save_new.json) for example).

It should be fixed with #6948.

**Comment by npracht** [comment]
Fixed by https://github.com/mautic/mautic/pull/6948 merged in 2.15.1
